Documentation/articles/getting_started/1_setting_up_your_development_environment_macos.md

```diff
@@ -7,7 +7,7 @@ Go to the following URL to download and install Visual Studio for Mac: https://v
 
 ## Install MonoGame extension for Visual Studio for Mac
 
-Download the MonoGame extension for Visual Studio for Mac from the following link: https://github.com/MonoGame/MonoGame/releases/download/v3.8/MonoDevelop.MonoGame_IDE_VisualStudioForMac_3.8.0.1641.mpack
+Download the MonoGame extension for Visual Studio for Mac from the following link: https://github.com/MonoGame/MonoGame/releases/download/v3.8/MonoDevelop.MonoGame_IDE_VisualStudioForMac_v3.8.0.1641-vsm8.0.mpack
```

Comment:
The link should point to v3.8.0.1643-vsm8.0.mpack

Reply:
The link on the [download page](https://www.monogame.net/downloads/) is for 1641 ("You can find the latest stable release here: MonoGame 3.8.0.1641"). Should that also be updated? That linked to a [forum page](https://community.monogame.net/t/monogame-3-8/13312) which linked to a [Github page](https://github.com/MonoGame/MonoGame/releases/tag/v3.8) which was for 1641.

I'm new to the C# ecosystem, so it was kind of hard for me to follow. Is there a way to automatically link to the latest stable build?

Reply:
We had to rebuild the extension with updated dependencies a month ago, so 1643 is still using MonoGame 3.8.0.1641, its just a bumped up version of the extension.

We could make some specific name to point to the latest release in the release, but what will happen at some point is switching the instructions to using the VS for Mac online extension gallery, as soon as Microsoft stops breaking it.